### PR TITLE
Added module-related functions to JIT

### PR DESCRIPTION
**Pull request by harlanhaskins**

### CODE REVIEW

File: Sources/LLVM/JIT.swift [hunk 1]
```diff
@@ -61,6 +67,40 @@ public final class JIT {
...
   }
 
+  /// Retrieves a pointer to the function compiled by this JIT.
+  /// - parameter name: The name of the function you wish to look up.
+  /// - returns: A pointer to the result of compiling the specified function.
+  /// - note: You will have to `unsafeBitCast` this pointer to
+  ///         the appropriate `@convention(c)` function type to be
+  ///         able to run it from Swift.
+  ///         
+  ///       typealias FnPtr = @convention(c) () -> Double
```
Comment (CodaFi) [inline Sources/LLVM/JIT.swift]: Code voice plz.

File: Sources/LLVM/JIT.swift [hunk 2]
```diff
@@ -61,6 +67,40 @@ public final class JIT {
...
+  ///       let fnAddr = jit.addressOfFunction(name: "test")
+  ///       let fn = unsafeBitCast(fnAddr, to: FnPtr.self)
+  public func addressOfFunction(name: String) -> OpaquePointer? {
```
Comment (CodaFi) [inline Sources/LLVM/JIT.swift:82]: Document the return type

Comment (harlanhaskins) [reply Sources/LLVM/JIT.swift:82]: What do you mean by document it, other than what's there?

Comment (CodaFi) [reply Sources/LLVM/JIT.swift:82]:
```
- returns:
```

Comment (harlanhaskins) [reply Sources/LLVM/JIT.swift:82]: It's there! 😅

**Review by CodaFi — APPROVED** [review]

**Pull request merged**